src/modules/simulator/simulator.h

```diff
@@ -265,76 +259,109 @@ class Simulator : public ModuleParams
 			delete _instance;
```

Comment:
I know it's unreleated, but this is really bad: this results in a recursive call of the destructor. You can just remove that (`simulator stop` isn't implemented properly, if you want to look into that).

Reply:
Thanks @bkueng !  I've pushed up a revised history to this PR so that it deals only with the issue you identified as well as the orb_unsubscribe() and perf_free() calls.  I'll put together another PR for renaming the methods to match convention in this file.

Thank you for your reviews!